doc/PAGES/MeshTutorials/MeshTutorial_07_3DExtrudedMesh.h

```diff
@@ -78,7 +78,7 @@ cylinder),
 -- Logical Volumes
 my_LV = LogicalVolumeCreate(RCC, 0, 0, 0.1, 0, 0, 0.2, 0.4)
```

Comment:
and here's an exmaple where we do not have module.function/action

Reply:
Personally, I would not focus too much on the header files in `doc`. I expect those to go away. I usually modify those for 2 reasons: (1) in case of salvaging the content, it will be closer to the current state, (2) when I do code-wide search, it does not show up any more, so less filtering on my side...

Reply:
True. I am assuming we will be removing the old syntax piece by piece as we move towards the new one?